app/routes/missions/icecube.mdx

```diff
@@ -39,7 +39,7 @@ signatures (cascades of ~m in extent) when they produce secondary electrons or h
 Track events can be reconstructed with an uncertainty of less than 1 degree, while cascade events
 have higher signal purity. IceCube triggers on signals for neutrino energies greater than ~10 GeV
 (10<sup>10</sup> eV) and can identify likely astrophysical neutrino events for neutrino
-energies greater than ~50 TeV (5 x 10<sup>13</sup> eV) and issue alerts to the community.
+energies greater than ~50 TeV (5x10<sup>13</sup> eV) and issue alerts to the community.
```

Comment:
That's the Unicode multiplication sign glyph, not the letter "x."

```suggestion
energies greater than ~50 TeV (5×10<sup>13</sup> eV) and issue alerts to the community.
```

Reply:
Apologies, missed that